java/ql/src/semmle/code/java/frameworks/spring/SpringHttp.qll

```diff
@@ -38,3 +39,21 @@ class SpringResponseEntityBodyBuilder extends Interface {
 class SpringHttpHeaders extends Class {
   SpringHttpHeaders() { this.hasQualifiedName("org.springframework.http", "HttpHeaders") }
 }
+
+// /** Models `org.springframework.http.RequestEntity` class. */
+// class SpringRequestEntity extends Class {
+//   SpringRequestEntity() { this.hasQualifiedName("org.springframework.http", "RequestEntity<>") }
+// }
+/** Models `org.springframework.http.RequestEntity`s instantiation expressions. */
+class SpringRequestEntityInstanceExpr extends ClassInstanceExpr {
+  int numArgs;
```

Comment:
Is this used anywhere?

Reply:
https://github.com/github/codeql/blob/a210fbbe69909e14c90f09ef77eb8f157e05ee4e/java/ql/src/experimental/CWE-918/RequestForgeryCustomizations.qll#L242